include/mfmg/dummy.hh

```diff
@@ -0,0 +1,18 @@
+/*************************************************************************
+ * Copyright (c) 2017 by the mfmg authors                                *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * This file is part of the mfmg libary. mfmg is distributed under a BSD *
+ * 3-clause license. For the licensing terms see the LICENSE file in the *
+ * top-level directory                                                   *
+ *************************************************************************/
```

Comment:
I recently read https://reuse.software/practices/. May not be a bad idea to follow that.

Reply:
I don't see what would be different with what we are doing now. Can you explain?

Reply:
It's not different in the spirit, just in details as the standardize the language and reference SPDX which allows for automated tracking of licenses.